src/algebra/big_operators/basic.lean

```diff
@@ -1570,6 +1574,15 @@ lemma prod_unique_nonempty {α β : Type*} [comm_monoid β] [unique α]
   (∏ x in s, f x) = f default :=
 by rw [h.eq_singleton_default, finset.prod_singleton]
 
+lemma sum_mod (s : finset α) {n : ℕ} (f : α → ℕ) : (∑ i in s, f i) % n = (∑ i in s, f i % n) % n :=
```

Comment:
I think it would be better to prove this for `list.sum`, then `multiset.sum`, then `finset.sum`.